src/components/dialogs/mras-search-info.vue

```diff
@@ -1,31 +1,32 @@
 <template>
   <v-dialog v-model="showModal" id="mras-search-info-modal" width="40rem" persistent>
     <v-card class="pa-9">
-      <v-row no-gutters>
-        <v-col cols="11">
-          <v-card-title>
-            Corporate Number Not Found
-          </v-card-title>
-        </v-col>
-        <v-col cols="1">
-          <v-icon md color="primary" @click="clearAndClose()">mdi-close</v-icon>
-        </v-col>
-      </v-row>
+      <v-card-title class="d-flex justify-space-between">
+        Corporate Number Not Found
+        <div class="mt-3">
+          <v-btn icon large class="dialog-close" @click="clearAndClose()">
+            <v-icon>mdi-close</v-icon>
+          </v-btn>
+        </div>
+      </v-card-title>
```

Comment:
Now using flexbox instead of v-row/v-col for layout. Looks like this:

![image](https://github.com/bcgov/namerequest/assets/10002889/da18f24d-46d6-4c02-8b8f-4e4c5fbcd7f3)

Reply:
What's the benefit here Sev? Doesn't it look the same as before?

Reply:
Before:

![image](https://github.com/bcgov/namerequest/assets/10002889/b89db0b3-f6ee-443b-b0a8-a67c1d59ea84)

In the new code, the X is better lined up. Overall, flexbox for this purpose (and correctly using dialog title, text, actions) makes for cleaner template code.

Reply:
Oh, OK. Got it.